TLM/TLM/Util/ModsCompatibilityChecker.cs

```diff
@@ -23,9 +24,11 @@ public class ModsCompatibilityChecker {
 
         // parsed contents of incompatible_mods.txt
         private readonly Dictionary<ulong, string> knownIncompatibleMods;
+        private readonly HashSet<string> gameBreaking;
```

Comment:
lol

Reply:
If having a mod installed that breaks XML parsing in every other mod then what is it? 😄

Reply:
pita xD
private readonly hashset<string> thisThingShouldDie =